electron/main/index.ts

```diff
@@ -4,6 +4,18 @@ import { join } from 'path';
 import { BitGoAPI } from '@bitgo/sdk-api';
 import { AbstractUtxoCoin, backupKeyRecovery } from '@bitgo/abstract-utxo';
 import { Btc, Tbtc } from '@bitgo/sdk-coin-btc';
+import { Eth, Gteth } from '@bitgo/sdk-coin-eth';
+import { Xlm, Txlm } from '@bitgo/sdk-coin-xlm';
+import { Xrp, Txrp } from '@bitgo/sdk-coin-xrp';
+import { Eos, Teos } from '@bitgo/sdk-coin-eos';
+import { Bch } from '@bitgo/sdk-coin-bch';
+import { Ltc } from '@bitgo/sdk-coin-ltc';
+import { Btg } from '@bitgo/sdk-coin-btg';
+import { Zec } from '@bitgo/sdk-coin-zec';
+import { Dash } from '@bitgo/sdk-coin-dash';
+import { Bcha } from '@bitgo/sdk-coin-bcha';
+import { Bsv } from '@bitgo/sdk-coin-bsv';
```

Comment:
do you not need the test variants of these?

Reply:
No